DataPlotly/core/plot_types/radar.py

```diff
@@ -0,0 +1,108 @@
+"""
+Radar chart factory
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+"""
+
+import os
+from plotly import graph_objs
+from qgis.PyQt.QtGui import QIcon
+from DataPlotly.core.plot_types.plot_type import PlotType
+import plotly.colors as pc
+
```

Comment:
missing QCoreApplication import:

from qgis.PyQt.QtCore import QCoreApplication

Reply:
woups

Reply:
fixed in ec08cd8101dffdbf403ee54821bc168adce0fbdd